packages/grafana-ui/src/components/Table/styles.ts

```diff
@@ -11,14 +11,28 @@ export function useTableStyles(theme: GrafanaTheme2, cellHeightOption: TableCell
   const rowHeight = cellHeight + 2;
   const headerHeight = 28;
 
-  const buildCellContainerStyle = (color?: string, background?: string, overflowOnHover?: boolean) => {
+  const buildCellContainerStyle = (
+    color?: string,
+    background?: string,
+    overflowOnHover?: boolean,
+    asCellText?: boolean
+  ) => {
     return css({
       label: overflowOnHover ? 'cellContainerOverflow' : 'cellContainerNoOverflow',
       padding: `${cellPadding}px`,
       width: '100%',
       // Cell height need to account for row border
       height: `${rowHeight - 1}px`,
-      display: 'flex',
+
+      display: asCellText ? 'block' : 'flex',
+
+      ...(asCellText ? {
+        overflow: 'hidden',
+        textOverflow: 'ellipsis',
+        userSelect: 'text',
+        whiteSpace: 'nowrap',
```

Comment:
this is copy/pasted from cellText class in this same file. when i tried to extract this to a constant for reuse, i got TS errors :(